### PR TITLE
Quartz updated to version 2.2.3

### PR DESCRIPTION
**Pull request by enricruiz**

### CODE REVIEW

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java [hunk 1]
```diff
@@ -223,12 +228,11 @@ protected Trigger createPeriodicTrigger(final String triggerName, final Integer
...
 
         // Create trigger
-        SimpleTrigger trigger = new SimpleTrigger(triggerName, DEFAULT_GROUP_NAME);
-        trigger.setStartTime(startTime.getTime());
-        trigger.setRepeatInterval(repeatInterval * repeatFactor);
-        trigger.setRepeatCount(SimpleTrigger.REPEAT_INDEFINITELY);
-
-        return trigger;
+        return TriggerBuilder.newTrigger() //
+            .withIdentity("licenseTrigger") //
```
Comment (nacx) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: Should you use `triggerName` instead?

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java [hunk 2]
```diff
@@ -124,7 +126,10 @@ public void schedule(final Class< ? > taskClass) throws TaskServiceException
         LOGGER.info("Adding task {} to {}", taskName, this.getClass().getSimpleName());
 
         // Create JobDetail Object
-        JobDetail jobDetail = new JobDetail(taskName, DEFAULT_GROUP_NAME, QuartzTask.class);
+        JobDetail jobDetail = JobBuilder.newJob(QuartzTask.class) //
+            .withIdentity(taskName) //
+            .build();
```
Comment (nacx) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: Are we no longer setting the jobs in a group?

Comment (enricruiz) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: By default the Jobs are scheduled in the DEFAULT_GROUP

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java (hunk 1 above)
Comment (enricruiz) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: Yes

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java (hunk 2 above)
Comment (nacx) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: Then remove the constant? Before the triggers where scheduled by default in the "abicloud" group. Would changing this have any side-effect?

Comment (enricruiz) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: I was wrong. I thought that the jobs was scheduled in quartz DEFAULT_GROUP not in a custom one. I will fix it.

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java [hunk 3]
```diff
@@ -245,9 +249,12 @@ protected Trigger createCronTrigger(final String triggerName, final String cronE
     {
         try
         {
-            return new CronTrigger(triggerName, DEFAULT_GROUP_NAME, cronExpression);
+            return TriggerBuilder.newTrigger() //
+                .withIdentity(triggerName) //
```
Comment (nacx) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: :rabbit:

File: src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java [hunk 4]
```diff
@@ -288,7 +295,9 @@ public void schedule(final Class< ? > taskClass, final int minutes) throws TaskS
         LOGGER.info("Adding task {} to {}", taskName, this.getClass().getSimpleName());
 
         // Create JobDetail Object
-        JobDetail jobDetail = new JobDetail(taskName, DEFAULT_GROUP_NAME, QuartzTask.class);
+        JobDetail jobDetail = JobBuilder.newJob(QuartzTask.class) //
+            .withIdentity(taskName) //
```
Comment (nacx) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: 🐰

Comment (enricruiz) [inline src/main/java/com/abiquo/taskservice/impl/quartz/QuartzTaskService.java]: :cry:

**Pull request merged**